### DISCUSSION TITLE
Multiple Volumes?

### DISCUSSION THREAD

**Discussion by tw333k** [jlesage/docker-mediainfo#8] [General]
Is there a way to add multiple storage volumes? I have media in different shared folders on my Synology NAS (Plex server) but can only access a single volume with Mediainfo.

**Comment by jlesage** [comment]
Yes, in your container's Advanced Settings, there is a tab name Volume where you can add additional volumes.

**Comment by tw333k** [comment]
sorry for the late reply. I've tried adding a second volume but when I navigate to it in the app, it's empty.
I'm not able to call it "storage" (I get an error saying it already exists) so I tried "storage2"
<img width="365" alt="image" src="https://github.com/user-attachments/assets/f574a6ce-2bc8-4ae9-a08b-3aef768ca7b9">
<img width="470" alt="image" src="https://github.com/user-attachments/assets/4744d14d-231b-4ff5-b422-00368a3e842c">
<img width="781" alt="image" src="https://github.com/user-attachments/assets/092f7ebf-e860-4cbf-bde3-89f9978ca8c1">

**Comment by tw333k** [comment]
Any suggestions?

**Reply by tw333k** [reply]
I guess it's currently not possible.
Any chance you can make it happen?

**Reply by jlesage** [reply]
If you don't see anything under `/storage2`, it might be that the container doesn't have the proper permissions. Make sure the permissions of files under this folder can be accessed by the user you configured for the container (`USER_ID`/`GROUP_ID`).

**Comment by tw333k** [comment]
the folder I'm trying to map to /storage2 is my Movies folder and the permissions are exactly the same as the folder I have mapped to /storage (TV folder)
UPDATE: the permissions were not the same. I had to set `Everyone` to read only and now it works!
Thanks man.

**Reply by jlesage** [reply]
Great!
